client/hotrod-client/src/test/java/org/infinispan/client/hotrod/retry/DistributionRetryTest.java

```diff
@@ -199,7 +183,7 @@ protected String parameters() {
    public Object[] factory() {
       return new Object[] {
             new DistributionRetryTest().withRetries(0),
-            new DistributionRetryTest().withRetries(10),
+//            new DistributionRetryTest().withRetries(10),
       };
```

Comment:
Should uncomment? This class was also flaky.

Reply:
Yeah, just adding back in.